src/main/java/com/nulabinc/zxcvbn/matchers/AlignedAdjacentAdjacentGraphBuilder.java

```diff
@@ -0,0 +1,37 @@
+package com.nulabinc.zxcvbn.matchers;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class AlignedAdjacentAdjacentGraphBuilder extends Keyboard.AdjacentGraphBuilder {
```

Comment:
This file and class name has been wrong for some time.

`AlignedAdjacentAdjacentGraphBuilder` -> `AlignedAdjacentGraphBuilder`

It's hard to review when there are more extra differences, so I can fix this with another issue.